calculator.js

```diff
@@ -0,0 +1,156 @@
+var calculator = {
+
+add:function (num1, num2){
```

Comment:
Watch your indentation! These should all be 1 tab from the left, showing they're inside `var calculator`